src/Frame.cc

```diff
@@ -76,11 +76,14 @@ Frame::Frame(const cv::Mat &imLeft, const cv::Mat &imRight, const double &timeSt
     mvLevelSigma2 = mpORBextractorLeft->GetScaleSigmaSquares();
     mvInvLevelSigma2 = mpORBextractorLeft->GetInverseScaleSigmaSquares();
 
+
+    ExtractORB(0,imLeft);
+    ExtractORB(1,imRight);
     // ORB extraction
-    thread threadLeft(&Frame::ExtractORB,this,0,imLeft);
+/*    thread threadLeft(&Frame::ExtractORB,this,0,imLeft);
     thread threadRight(&Frame::ExtractORB,this,1,imRight);
     threadLeft.join();
-    threadRight.join();
+    threadRight.join();*/
```

Comment:
Why to remove the threading code?

Reply:
Now std::thread error on exit https://github.com/kripken/emscripten/issues/3721

Reply:
Thanks! Please add this info as comment. And please replace code instead of commenting them out. It's better to see the diff.